lesson07/assignment/assignment07/src/App.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div id="app">
```

Comment:
As this is the content that will be inserted into the HTML in index.html, we don't want to duplicate the id of "app".